tests/testKalturaFFMpegCmdGenerator.js

```diff
@@ -0,0 +1,55 @@
+
+const chai = require('chai');
+const expect = chai.expect;
+const KalturaFFMpegCmdGenerator = require('../lib/utils/KalturaFFMpegCmdGenerator');
+const kalturaTypes = require('../lib/client/KalturaTypes');
+const ApiClientConnector = require('../lib/infra/ApiServerClientConnector');
+const KalturaMediaInfo = require('../lib/utils/KalturaMediaInfo');
+const KalturaMediaInfoResponse = require('../lib/utils/KalturaMediaInfoResponse');
+
+describe('test KalturaFFMpegCmdGenerator', function() {
+
+    let serviceUrl = KalturaConfig.config.testClient.serviceUrl;
+    let secret = KalturaConfig.config.testClient.secret;
+    let partnerId = KalturaConfig.config.testClient.partnerId;
+    let testsDirName = __dirname;
+    let flavorId = '0_t50woq53';

```

Comment:
saw you created a testing.ini - couldn't we set it there - this for sure will not work with all envs